sympy/matrices/expressions/matexpr.py

```diff
@@ -1026,6 +1035,44 @@ def _entry(self, i, j, **kwargs):
         return S.One
 
 
+class SingleEntryMatrix(MatrixExpr):
+    """A matrix which is zero everywhere except the entry (i, j) which
```

Comment:
"A m x n matrix"

Also this needs an example to show what the argument order is.